core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchReaderIterator.java

```diff
@@ -274,11 +281,13 @@ private void binRanges(TabletLocator tabletLocator, List<Range> ranges,
               failures.size());
         }
 
+        retry.useRetry();
```

Comment:
Not a a problem with this PR, but I have not been calling this.   I looked at the code and it will influence the message logged by waitForNextAttempt

Reply:
Removed in 65b79ae